docs/rules/prefer-expect-query-by.md

```diff
@@ -24,12 +26,14 @@ test('some test', () => {
 ```
 
 ```js
-test('some test', () => {
-  const rendered = render(<App />);
-  expect(rendered.getByText('Foo')).toBeInTheDocument();
-  expect(rendered.getAllByText('Foo')[0]).toBeInTheDocument();
```

Comment:
Should we also keep the examples without `not`? At the end it's still valid to use `expect(queryByText('Foo')).toBeInTheDocument()`

Reply:
Of course 🤦‍♂️. Let me get this back.

Reply:
Fixed